Code/Engine/Foundation/Strings/PathUtils.h

```diff
@@ -25,31 +26,31 @@ class EZ_FOUNDATION_DLL ezPathUtils
   /// \brief Checks every character in the string with IsValidFilenameChar()
   ///
   /// This is a basic check, only because each character passes the test, it does not guarantee that the full string is a valid path.
-  static bool ContainsInvalidFilenameChars(const char* szPath, const char* szPathEnd = ezMaxStringEnd);
+  static bool ContainsInvalidFilenameChars(const char* szPath, const char* szPathEnd = ezUnicodeUtils::GetMaxStringEnd<char>());
```

Comment:
why don't you use the stringutil variant here? looks irritating to use the unicodeutils thing here